api/v1alpha1/externaldns_types.go

```diff
@@ -442,8 +445,10 @@ type ExternalDNSSourceUnion struct {
 	// +required
 	Type ExternalDNSSourceType `json:"type"`
 
-	// LabelFilter specifies a label selector used to filter which source instance resources ExternalDNS publishes
-	// records for. The filter uses label selector semantics against source resource labels.
+	// LabelFilter specifies a label selector used to filter which source
+	// instance resources ExternalDNS publishes records for. The filter uses
+	// label selector semantics against source resource labels.  A null or
+	// empty label selector selects all objects with matching kind.
```

Comment:
Can someone double-check that this text is accurate?

Reply:
Not sure this is better? just some ideas
```suggestion
	//  LabelFilter specifies a label selector used to filter which instance(s) 
	// of the specified source type ExternalDNS publishes records for.
	// The filter uses label selector semantics matched against labels defined on 
	// instances of the specified source type.  A null or empty label selector 
	// selects all objects with matching source type.
```